build.sbt

```diff
@@ -7,29 +7,37 @@ description:= "handle outbound messages from salesforce to update zuora and iden
 
 version := "1.0"
 
-scalaVersion := "2.12.15"
+scalaVersion := "2.12.19"
 
 scalacOptions ++= Seq(
   "-deprecation",
   "-encoding", "UTF-8",
-  "-target:jvm-1.8",
+  "-release:8",
   "-Ywarn-dead-code"
 )
 
+enablePlugins(ScalaxbPlugin)
+
+Compile / scalaxb / scalaxbPackageName := "salesforce.soap"
+Compile / scalaxb / scalaxbGenerateDispatchClient := false // we don't need to use the 'dispatch' library
```

Comment:
`scalaxbGenerateDispatchClient` defaults to true, and generates an HTTP client for the SOAP service using the `dispatch` library, but we don't need an HTTP client, so it's cleaner to turn that off, and not have to pull in a `dispatch` dependency.